ui/page/seedbackup/verify_seed.go

```diff
@@ -198,26 +207,23 @@ func (pg *VerifySeedPage) Layout(gtx C) D {
 			promptToExit(pg.Load)
 		},
 		Body: func(gtx C) D {
-			wdg := []layout.Widget{
-				func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
 					label := pg.Theme.Label(values.TextSize16, "Select the correct words to verify.")
 					label.Color = pg.Theme.Color.Gray3
 					return label.Layout(gtx)
-				},
-				func(gtx C) D {
+				}),
+				layout.Rigid(func(gtx C) D {
 					return layout.Inset{
 						Bottom: values.MarginPadding96,
 					}.Layout(gtx, func(gtx C) D {
-						return pg.seedList.Layout(gtx, len(pg.multiSeedList), func(gtx C, index int) D {
-							return pg.seedListRow(gtx, index, pg.multiSeedList[index])
+						gtx.Constraints.Min.X = gtx.Constraints.Max.X
+						return material.List(pg.Theme.Base, pg.list).Layout(gtx, len(pg.multiSeedList), func(gtx C, i int) D {
```

Comment:
Why are we using `material.List`? Kindly add this as a file on the decredmaterial package. list.go.
See here for examples.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/checkbox.go.
https://github.com/planetdecred/godcr/blob/master/ui/decredmaterial/label.go.